packages/python-packages/api-stub-generator/apistub/_apiview.py

```diff
@@ -202,6 +204,12 @@ def add_literal(self, value):
     def add_navigation(self, navigation):
         self.navigation.append(navigation)
 
+    def _cross_language_id(self, namespace, name):
+        if name.startswith(namespace):
```

Comment:
You may want to check for a separator as well (e.g. '.')

Reply:
Actually, in the update, we don't strip anything. The key will just be the fully-qualified ID.

Reply:
Is this scenario possible?

```python
self._cross_language_id(namespace="a", name="abc.def")
```

Reply:
It would be a logic bug, but now I don't strip the namespace. I originally did as a convenience but I figured it's better to keep it simple and autorest can generate the right IDs anyhow.

Reply:
Ah, should have checked your update code.